routes/api/user.js

```diff
@@ -15,27 +17,58 @@ router.post('/register', async function(req, res, next) {
     email: req.body.email,
```

Comment:
How about trimming and lowercasing the user input?

Reply:
Validations are not implemented now. Need to implement for all routes with Joi later.